components/molecules/AuthContentWrapper/auth-content-wrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import clsx from "clsx";
+
+import Button from "components/atoms/Button/button";
+import Icon from "components/atoms/Icon/icon";
+
+import GitHubIcon from "img/icons/github-icon.svg";
+
+import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
+
+interface AuthContentWrapperProps {
+  children: React.ReactNode;
+}
+
+const AuthContentWrapper: React.FC<AuthContentWrapperProps> = ({ children }) => {
+  const router = useRouter();
+  const currentPath = router.asPath;
+
+  const { signIn, user: isAuthed } = useSupabaseAuth();
+
+  const [host, setHost] = useState<string>("");
```

Comment:
nit: the type is inferred automatically in this case
```suggestion
  const [host, setHost] = useState("");
```